src/actor/Claimer.spec.ts

```diff
@@ -296,4 +302,103 @@ describe('Claimer', () => {
     )
     expect(Array.isArray(presentation.body.content))
   })
+  describe('Negative tests', () => {
+    describe('create presentation', () => {
+      // fakes the input for Claimer.createPresentation
+      const fakePresentation = ({
+        messageBody = MessageBodyType.REQUEST_CLAIMS_FOR_CTYPES,
+        allowPE = false,
+        credentials = [],
+      }: {
+        messageBody?: MessageBodyType
+        allowPE?: boolean
+        credentials?: Credential[]
+      }): Promise<Message> => {
+        return Claimer.createPresentation(
+          claimer,
+          ({
+            body: {
+              type: messageBody, // should be default value
+              content: { allowPE },
+            },
+          } as unknown) as Message,
+          verifier.getPublicIdentity(),
+          credentials,
+          [],
+          true
+        )
+      }
+      it('Should throw when message body type does not match', () => {
+        return expect(
+          fakePresentation({
+            messageBody: MessageBodyType.REJECT_ATTESTATION_FOR_CLAIM,
+          })
+        ).rejects.toThrowError(
+          ERROR_MESSAGE_TYPE(
+            MessageBodyType.REJECT_ATTESTATION_FOR_CLAIM,
+            MessageBodyType.REQUEST_CLAIMS_FOR_CTYPES
+          )
+        )
+      })
+      it('Should throw when PE is required but not allowed', () => {
+        return expect(fakePresentation({})).rejects.toThrowError(
+          ERROR_PE_MISMATCH()
+        )
+      })
+      it('Should throw when PE is allowed and required but credentials have nulls', () => {
+        return expect(
+          fakePresentation({
+            allowPE: true,
+            credentials: [
+              ({ privacyCredential: null } as unknown) as Credential,
+            ],
+          })
+        ).rejects.toThrowError(ERROR_PE_CREDENTIAL_MISSING())
+      })
+      it('Should throw when PE is required but not allowed', () => {
+        return expect(fakePresentation({})).rejects.toThrowError(
+          ERROR_PE_MISMATCH()
+        )
+      })
```

Comment:
this looks like a duplicate from line 343++

Reply:
Oops, you are correct.

Reply:
Fixed in 5db18c9